src/main/webapp/pages/projects/project_samples_metadata_upload.html

```diff
@@ -1,132 +1,18 @@
 <!DOCTYPE html>
 <html
-      xmlns:th="http://www.thymeleaf.org"
-      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
-      data-layout-decorate="~{projects/_base}">
```

Comment:
Inside your root tag on this page, can you add a comment about the path to the base javascript file.

Reply:
Please see 0084bfc44642de61afea58b83d4d6591465de280.